src/routes/group/new/+page.svelte

```diff
@@ -58,115 +60,133 @@
 	{#if $chats.loading}
-		<Container align="center" grow gap={6} justify="center">
-			<div class="center">
-				<h2>Loading...</h2>
-			</div>
-		</Container>
+		<Layout>
+			<Container align="center" gap={6} justify="center">
+				<div class="center">
+					<h2>Loading...</h2>
+				</div>
+			</Container>
+		</Layout>
 	{:else if $chats.chats.size === 0}
 		{@const address = wallet.address}
-		<Header title="Create group">
-			<svelte:fragment slot="left">
-				<div class="header-btns">
-					<Button variant="icon" on:click={() => history.go(-1)}>
-						<ChevronLeft />
-					</Button>
-				</div>
+		<Layout>
+			<svelte:fragment slot="header">
+				<Header title="Create group">
+					<svelte:fragment slot="left">
+						<div class="header-btns">
+							<Button variant="icon" on:click={() => history.go(-1)}>
+								<ChevronLeft />
+							</Button>
+						</div>
+					</svelte:fragment>
+				</Header>
 			</svelte:fragment>
-		</Header>
-		<Container align="center" grow gap={6} justify="center" padX={24}>
-			<p class="text-lg text-bold">No contacts</p>
-			<p class="text-lg">You can only create groups from existing chat contacts</p>
-			<div class="btn-spacing">
-				<Button on:click={() => goto(routes.INVITE(address))}>
-					<AddComment />
-					Invite to chat
-				</Button>
-			</div>
-		</Container>
-	{:else if screen === 'create'}
-		<Header title="Create group">
-			<svelte:fragment slot="left">
-				<div class="header-btns">
-					<Button variant="icon" on:click={() => history.go(-1)}>
-						<ChevronLeft />
+			<Container align="center" gap={6} justify="center" padX={24}>
+				<p class="text-lg text-bold">No contacts</p>
+				<p class="text-lg">You can only create groups from existing chat contacts</p>
+				<div class="btn-spacing">
+					<Button on:click={() => goto(routes.INVITE(address))}>
+						<AddComment />
+						Invite to chat
 					</Button>
 				</div>
+			</Container>
+		</Layout>
+	{:else if screen === 'create'}
+		<Layout>
+			<svelte:fragment slot="header">
+				<Header title="Create group">
+					<svelte:fragment slot="left">
+						<div class="header-btns">
+							<Button variant="icon" on:click={() => history.go(-1)}>
+								<ChevronLeft />
+							</Button>
+						</div>
+					</svelte:fragment>
+				</Header>
 			</svelte:fragment>
-		</Header>
-		<ul class="chats" aria-label="Contact List">
-			{#each [...$chats.chats] as [id, chat]}
-				{#if !isGroupChatId(id)}
-					<li>
-						<div class="chat-button" role="listitem">
-							<Container grow>
-								<div class="chat">
-									<Avatar size={70} picture={chat.users[0].avatar} />
-									<div class="content">
-										<div class="user-info">
-											<span class="username text-lg text-bold">
-												{chat.users[0].name}
-											</span>
+			<ul class="chats" aria-label="Contact List">
+				{#each [...$chats.chats] as [id, chat]}
+					{#if !isGroupChatId(id)}
+						<li>
+							<!-- svelte-ignore a11y-label-has-associated-control -->
```

Comment:
Can we get rid of this warning? 
There is an input element, it's just nested inside a component.